akka-typed/src/main/scala/akka/typed/persistence/scaladsl/PersistentActor.scala

```diff
@@ -41,19 +41,26 @@ object PersistentActor {
    * Factories for effects - how a persitent actor reacts on a command
    */
   object Effect {
-    def persist[Event, State](event: Event): Effect[Event, State] =
-      new Persist[Event, State](event)
 
-    def persistAll[Event, State](events: im.Seq[Event]): Effect[Event, State] =
+    def persist[Event, State](event: Event, events: Event*): Effect[Event, State] =
+      if (events.nonEmpty)
+        Effect.persist(im.Seq(event) ++ events)
```

Comment:
Maybe spell it out that this is prepending to a `List` rather than have that hidden in the std-library? `event :: events.toList`

Reply:
Fair enough, I will change that. 

I'm also investigating if it makes sense to remove `Persist` and keep only `PersistAll`. That would remove the if/else check. 
The only drawback I see is the allocation of a `Seq` with only one single event. Is that a real issue?